src/NserviceBus.Transport.Msmq/MsmqTimeToBeReceivedOverrideCheck.cs

```diff
@@ -19,8 +19,19 @@ public StartupCheckResult CheckTimeToBeReceivedOverrides()
             var isTransactional = settings.GetRequiredTransactionModeForReceives() != TransportTransactionMode.None;
             var outBoxRunning = settings.IsFeatureActive(typeof(Features.Outbox));
 
-            var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
-            var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
+            //var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //workaround for AuditConfigReader.Result not being accessible
```

Comment:
So my fancy `dynamic` workaround didn't actual work, because `dynamic` still respects visibility. So even though I can see the property on the object, it wouldn't let me access it. 😢